Index: app/helpers/application_helper.rb
===================================================================
--- app/helpers/application_helper.rb	(revision 43)
+++ app/helpers/application_helper.rb	(working copy)
@@ -18,10 +18,11 @@
     link_to date, { :controller => 'game', :action => 'list', :id => date}
   end
 
-  def stat_header
+  def stat_header(name_map = {})
     result = %{<tr>\n<th class="statheadcell">Team</th>\n}
     for column in TeamAverage.stat_columns do
-      result += %{<th class="statheadcell">#{column.humanize}</th>}
+      name = name_map[column] ? name_map[column] : column.humanize
+      result += %{<th class="statheadcell">#{name}</th>}
     end
     result += "</tr>\n"
   end
Index: app/models/team_average.rb
===================================================================
--- app/models/team_average.rb	(revision 43)
+++ app/models/team_average.rb	(working copy)
@@ -2,8 +2,9 @@
   belongs_to :team
 
   def self.stat_columns
-    %w(half1_point half2_point total_point fgm fga fgp tpm tpa tpp 
-       ftm fta ftp offense_rebound total_rebound assist steal block turnover foul)
+    %w(total_point fgm fga fgp tpm tpa tpp 
+       ftm fta ftp offense_rebound total_rebound assist steal block turnover foul
+       ppp get_ft orp to_rate poss)
   end
   def self.percent_hash
     %w(fgp tpp ftp).inject({}) {|h,w| h[w] = true; h}
Index: app/views/team_stats/list.rhtml
===================================================================
--- app/views/team_stats/list.rhtml	(revision 43)
+++ app/views/team_stats/list.rhtml	(working copy)
@@ -1,7 +1,7 @@
 <h1>Team <%= @foe == '1' ? 'Opponent' : @foe == 'diff' ? 'Difference' : '' %> statistics overview</h1>
 
 <table class="sortable" id="statidunique">
-  <%= stat_header %>
+  <%= stat_header(@name_map) %>
 <% @team_stats.each do |s| -%>
 <%   team = @teams[s.team_id] -%>
   <tr class="<%= team.in_64 ? 'in64' : '' %>stat">
 
 
